src/Components/RosTable/DiskUsage.js

```diff
@@ -0,0 +1,48 @@
+import { Tooltip } from '@patternfly/react-core';
+import React from 'react';
+import { NO_DATA_STATE, NO_DATA_VALUE } from '../../constants';
+import './DiskUsage.scss';
+
+export const diskUsageTitle = () =>{
+    return (
+        <Tooltip content={<span>IOPS</span>}>
+            <span>I/O utilization</span>
+        </Tooltip>
+    );
+};
+
+export const DiskUsageData = (data, id, item) => {
+    const { state, performance_utilization: performanceUtilization } = item;
+    const { io_all: iopsAll } = performanceUtilization;
+
+    return (
+        state === NO_DATA_STATE ?
+            <span>{ NO_DATA_VALUE }</span> :
+            <Tooltip position="right" content={
+                <div>
+                    <table>
```

Comment:
Can you make use of patternfly `compactBorderless` table component - https://www.patternfly.org/v4/components/table#composable-basic?

Reply:
Thanks for sharing @kgaikwad it is okay if I will address it in separate PR?